docs/HowTo/Get-Started/Use-External-Signer.md

```diff
@@ -0,0 +1,37 @@
+---
+description: Use Eth2Signer for signing
+---
+
+# Use an external signer
+
+Teku supports the [Eth2Signer] external signing client.
+
+**Prerequisites**:
+
+* Eth2Signer installed and running
+* Configured signing key configuration files
+
+## Start Teku
+
+Start Teku and specify the external signer options.
+
+```bash
+teku --eth1-deposit-contract-address=dddddddddddddddddddddddddddddddddddddddd \
+--eth1-endpoint=http://localhost:8545 --validators-key-file=validator_keys \
+--validators-external-signer-public-keys=0xa99a...e44c,0xb89b...4a0b \
+--validators-external-signer-url=http://localhost:9000
+```
+
+The command line:
+
+* Specifies the validator public keys for which Eth2Signer signs attestations and blocks using
+    [`--validators-external-signer-public-keys`](../../Reference/CLI/CLI-Syntax.md#validators-external-signer-public-keys).
+* Specifies the url of the Eth2Signer client using
+    [`--validators-external-signer-url`](../../Reference/CLI/CLI-Syntax.md#validators-external-signer-url).
+
+!!! note
+    Create a signing key configuration file for each public key specified using
```

Comment:
Not sure what is meant by this. Are you referring to the configuration file in Eth2Signer that is needed for each key?

Reply:
Yes I am. Will try and rephrase this to make more clear.

Reply:
Will link to the appropriate Eth2Signer content when that content is merged.